src/frontend/screens/Settings/MapSettings/OfflineAreas.tsx

```diff
@@ -0,0 +1,179 @@
+import { BottomSheetModalMethods } from "@gorhom/bottom-sheet/lib/typescript/types";
+import * as React from "react";
+import { defineMessages, FormattedMessage, useIntl } from "react-intl";
+import { ScrollView, StyleSheet } from "react-native";
+import { NavigationStackScreenComponent } from "react-navigation-stack";
+
+import { RED } from "../../../lib/styles";
+import {
+  BottomSheetContent,
+  BottomSheetModal,
+  useBottomSheetModal,
+} from "../../../sharedComponents/BottomSheetModal";
+import Button from "../../../sharedComponents/Button";
+import HeaderTitle from "../../../sharedComponents/HeaderTitle";
+import { ErrorIcon } from "../../../sharedComponents/icons";
+import Loading from "../../../sharedComponents/Loading";
+import { OfflineAreaCard } from "../../../sharedComponents/OfflineAreaCard";
+
+const m = defineMessages({
+  title: {
+    id: "screens.Settings.MapSettings.OfflineArea",
+    defaultMessage: "Offline Areas",
+  },
+  removeMap: {
+    id: "screens.Settings.MapSettings.removeMap",
+    defaultMessage: "Remove Map",
+  },
+  cancel: {
+    id: "screens.Settings.MapSettings.cancel",
+    defaultMessage: "Cancel",
+  },
+  clearDiagrams: {
+    id: "screens.Settings.MapSettings.clearDiagrams",
+    defaultMessage: "Are you sure you want to Clear diagrams",
+  },
+  subtitle: {
+    id: "screens.Settings.MapSettings.subtitle",
+    defaultMessage:
+      "This map and offline areas attached to it will be deleted. This cannot be undone",
```

Comment:
```suggestion
      "This map and offline areas attached to it will be deleted. This cannot be undone.",
```

Reply:
not entirely sure about the use of "attached" here but i think it's generally fine. just wanted to point out in case others had thoughts

Reply:
This is from the copy deck

Reply:
yeah i'm aware of that, but my comment still stands 😄